store/map/index.js

```diff
@@ -140,20 +117,53 @@ export const actions = {
           .format('YYYY-MM-DDTHH:mm:ssZ'),
         datasetId: datasetId,
       }
+      getFromApi('timeseries', parameters).then(response => {
+        const pointDataType = _.get(
+          state.datasets[datasetId].metadata,
+          'pointData',
+        )
 
-      return getFromApi('timeseries', parameters).then(({ results }) => {
-        commit(
-          'datasets/addDatasetPointData',
-          Object.freeze({
-            id: datasetId,
-            data: {
-              [locationId]: {
-                category: getFormattedTimeStamps(results),
-                serie: getValues(results),
+        if (pointDataType === 'images') {
+          commit(
+            'datasets/addDatasetPointData',
+            Object.freeze({
+              id: datasetId,
+              data: {
+                [locationId]: {
+                  imageUrl: response,
+                },
               },
-            },
-          }),
-        )
+            }),
+          )
+        } else {
+          let category = []
+          let serie = []
+          const eventResults = response.results.filter(res =>
+            _.has(res, 'events'),
+          )
+
+          eventResults.forEach(res => {
+            serie = serie.concat(res.events.map(event => event.value))
+            category = category.concat(
+              res.events.map(event =>
+                moment(event.timeStamp).format('MM-DD-YYYY HH:mm'),
```

Comment:
what's with the , at the end of the line?

Reply:
linter settings..